src/focus-trap-react.js

```diff
@@ -42,6 +42,18 @@ class FocusTrap extends React.Component {
         continue;
       }
 
+      /**
+       * If a boolean value is passed to 'allowOutsideClick',
+       * wrap it in a function that returns that value
+       */
+      if (
+        optionName === 'allowOutsideClick' &&
+        typeof specifiedFocusTrapOptions[optionName] === 'boolean'
+      ) {
+        tailoredFocusTrapOptions[optionName] = () =>
+          specifiedFocusTrapOptions[optionName];
+      }
+
       tailoredFocusTrapOptions[optionName] =
         specifiedFocusTrapOptions[optionName];
```

Comment:
I think we need to put lines 57-58 in an `else` block here. Otherwise, they will just overwrite what we're now doing in lines 53-54, won't they? The function we just set will be overwritten by the boolean we attempted to special-case.

Reply:
Good catch! I'd like to think that the tests that I wanted to right would have caught that 😂